### DISCUSSION TITLE
Sharing palettes in 5.0.0

### DISCUSSION THREAD

**Discussion by pfrankov** [pfrankov/obsidian-colored-tags#18] [Show and tell]
Making a palette is easy:

1. Go to online generator (e.g. https://coolors.co, https://colorkit.co) and pick whatever you want
2. Your URL-address contains the palette in almost right format: `https://coolors.co/1b065e-ff47da-ff87ab-fcc8c2-f5eccd`
3. Just take everything after the last `/` and paste it `1b065e-ff47da-ff87ab-fcc8c2-f5eccd` in custom palette field in the plugin

Share the best.
There's a limit of one palette per user; additional ones will be ignored.
Use reactions to upvote or downvote, not the default ↑ button.

**Comment by pfrankov** [comment]
<img width="553" alt="image" src="https://github.com/pfrankov/obsidian-colored-tags/assets/584632/b4dcf83e-aa4e-4bd8-a1f0-53d5e6ce69a4">

```
e12729-f37324-f8cc1b-72b043-a07f4e
```

**Comment by dhniceday** [comment]
![Bildschirmfoto vom 2024-01-07 23-19-27](https://github.com/pfrankov/obsidian-colored-tags/assets/3727327/be018f14-09a1-40b4-944c-24a91e6c1f9d)

```
5ccfe6-ffd173-d5ff80-95e6cb-f28799-ffad66-ffdfb3-dfbfff-f29e74-87d96c-80bfff-f27983-cccac2
```

**Reply by pfrankov** [reply]
Thank you for the first custom palette!
I've edited to code block to make it easy to copy.

**Reply by pfrankov** [reply]
@myhd Thank you! Could you move your palette to the main thread of discussion?

**Reply by shmbr** [reply]
A bit darker version

<img width="771" alt="Screenshot 2025-03-15 at 15 06 15" src="https://github.com/user-attachments/assets/c293bf0b-ea4f-4678-80d6-866681823971" />


```
0F4A5A-7A501F-526A2D-284A46-5A1F2D-6A3A13-6A4D2F-4F3A7A-4F2D1F-1F3D1A-1A3A5A-4F1A2D-3D3D3D
```

**Comment by kimthompson** [comment]
My beloved [Catppuccin Mocha](https://github.com/catppuccin/catppuccin):

![Screenshot 2024-03-05 at 9 59 33 AM](https://github.com/pfrankov/obsidian-colored-tags/assets/3376722/cda76e79-92d3-48d0-bb5d-a277ddca8bce)
```
f38ba8-fab387-f9e2af-a6e3a1-89b4fa-cba6f7
```

**Reply by kimthompson** [reply]
I actually ended up using the Latte (light) flavor for bolder tag colors:

<img width="765" alt="Screenshot 2024-03-05 at 10 24 51 AM" src="https://github.com/pfrankov/obsidian-colored-tags/assets/3376722/d24fa3e0-29fc-45c8-94b2-dc089d1a9451">

```
d20f39-fe640b-df8e1d-40a02b-1e66f5-8839ef
```

**Comment by N1cknamed** [comment]
Just the accent colours from Tokyo Night to use with the theme. I recommend enabling high contrast text with these.

![image](https://github.com/pfrankov/obsidian-colored-tags/assets/19593132/d6648822-8250-4b84-8fdd-d25ac5d221d7)


`ff757f-e0af68-9ece6a-7dcfff-7aa2f7-bb9af7`

**Comment by sdrwtf** [comment]
Here are my five cents:

![image](https://github.com/pfrankov/obsidian-colored-tags/assets/34947398/109f9168-748d-4fff-b51f-5950cad279f4)

```
F94144-F3722C-F8961E-F9C74F-90BE6D-43AA8B-4D908E-577590-277DA1
``` 

`Palette shift` set to `0`

**Comment by jwklong** [comment]
repost because i accidentally commented on somebody else's pallete
```
962637-965318-8c6f1f-408534-298a5e-217680-216496-1c2c85-431f8f-6920a1-8c2091-a32274-a3225e
```
![image](https://github.com/pfrankov/obsidian-colored-tags/assets/72522395/f09362f4-070c-4821-9b41-006b7f710078)

**Comment by ELowry** [comment]
FFD918-FFA635-FF7252-FF3F6F-C63273-8D2577-54187B
![image](https://github.com/pfrankov/obsidian-colored-tags/assets/7792033/941bd95a-3103-4755-8405-d1a103bc828c)

**Comment by AndreaC-7** [comment]
One for us [Dracula Theme](https://draculatheme.com/) aficionados:

<img width="593" alt="colorkit" src="https://github.com/pfrankov/obsidian-colored-tags/assets/73082483/dabd0dd9-9de9-41c4-8e34-9a441f4999ea">

```
44475A-F8F8F2-6272A4-8BE9FD-50FA7B-FFB86C-FF79C6-BD93F9-FF5555-F1FA8C
```

**Reply by cyrus01337** [reply]
Hell yeah!

**Comment by myhd** [comment]
### Visually Equidistant Tag Colors

Created with [I want Hue (Colors for data scientists)](https://medialab.github.io/iwanthue/) #wordplay

<img width="777" alt="image" src="https://github.com/pfrankov/obsidian-colored-tags/assets/1010447/b664a01f-4142-45ec-84e3-4b00bd90391b">

 ```
9550a1-ba563c-687ad2-a09e3f-50b47a
```

**Comment by mukiwu** [comment]
<img width="626" alt="截圖 2024-08-21 13 24 23" src="https://github.com/user-attachments/assets/9ee92e76-25bf-42a6-bc8d-a62e886e9653">

```
95B2B1-A0C1D1-9F9AA4-D1A49F-E7CFCD-AFC2D5-CFD8D7-B5C9C3-CAB1BD
```

**Comment by rareyman** [comment]
### Nord Palette

<img width="780" alt="image" src="https://github.com/user-attachments/assets/7050d6bf-3a76-4985-a2fd-4a43a125d631">

```
a3be8c-eccc8c-d08871-c0616a-b48fad-88c1d0-82a1c1-5e81ad-4c566a-2e3440
```
(Minor edit to the order)

**Comment by inhumansigma** [comment]
![image](https://github.com/user-attachments/assets/8ba19f3a-93bf-48ff-9bdf-a410eadb0402)

`337BFF-4B0082-800080-FF33DB-FF337B-FF3333-FF6633-FF9933-FFCC33-DBFF33-9BFF3D-33FFCD`

**Comment by inhumansigma** [comment]
![image](https://github.com/user-attachments/assets/6b487291-ebb7-491e-9dd3-30df92b8f554)

```000000-0D0D0D-1A1A1A-262626-333333-404040-4D4D4D-595959-666666-737373-808080-8C8C8C-999999-A6A6A6-B3B3B3-C0C0C0-CDCDCD-D9D9D9-E6E6E6-FFFFFF FF0000-FF0C00-FF1900-FF2500-FF3200-FF3E00-FF4A00-FF5700-FF6400-FF7000-FF7D00-FF8A00-FF9700-FFA300-FFB000-FFBC00-FFC800-FFD500-FFD800-FFFF00 DBFF00-B9FF00-97FF00-77FF00-55FF00-33FF00-00FF00-00FF1A-00FF33-00FF4C-00FF66-00FF80-00FF99-00FFB2-00FFCC-00FFFF-00D1FF-00A3FF-0075FF-0047FF-0019FF-0000FF 1A00FF-3B00FF-5D00FF-7F00FF-9F00FF-B200FF-D500FF-FF00E0-FF00CC-FF00B2-FF0099-FF007F-FF0066-FF0050-FF0033-FF0019-FF000C```

**Reply by SandmansDreams** [reply]
Wish this worked

**Reply by 3hr80FEZwlir** [reply]
Doesn't work.

**Reply by SierraKomodo** [reply]
It had several cases of spaces instead of dashes. Here's a fixed version.

```
000000-0D0D0D-1A1A1A-262626-333333-404040-4D4D4D-595959-666666-737373-808080-8C8C8C-999999-A6A6A6-B3B3B3-C0C0C0-CDCDCD-D9D9D9-E6E6E6-FFFFFF-FF0000-FF0C00-FF1900-FF2500-FF3200-FF3E00-FF4A00-FF5700-FF6400-FF7000-FF7D00-FF8A00-FF9700-FFA300-FFB000-FFBC00-FFC800-FFD500-FFD800-FFFF00-DBFF00-B9FF00-97FF00-77FF00-55FF00-33FF00-00FF00-00FF1A-00FF33-00FF4C-00FF66-00FF80-00FF99-00FFB2-00FFCC-00FFFF-00D1FF-00A3FF-0075FF-0047FF-0019FF-0000FF-1A00FF-3B00FF-5D00FF-7F00FF-9F00FF-B200FF-D500FF-FF00E0-FF00CC-FF00B2-FF0099-FF007F-FF0066-FF0050-FF0033-FF0019-FF000C
```

**Comment by MayADevBe** [comment]
## [Gruvbox](https://github.com/morhetz/gruvbox) Dark Colors

<img width="496" alt="gruvboxdark" src="https://github.com/user-attachments/assets/4c5d9948-cd32-47ad-948c-e4c194eab2f2">

```a3be8c-eccc8c-d08871-c0616a-b48fad-88c1d0-82a1c1-5e81ad-4c566a-2e3440```

**Comment by championswimmer** [comment]
Light Shades 
<img width="775" alt="image" src="https://github.com/user-attachments/assets/0bd1aac7-79fa-41eb-94e4-da2c7b1abef8" />

```
bffff0-ffe0e2-ffd29c-e5ff9c-bfd4ff-bfffc1-decfff-ffcffa
```

Light Shades V2 

<img width="773" alt="image" src="https://github.com/user-attachments/assets/e7cd1fe7-9d9a-4d1d-89e3-3dd62cf4ee71" />


```
e0c2f1-ecc2e4-f8c1d7-fec7cd-fad8ca-f5e9c7-eff9c3-e6ffca-daffd7-ceffe4
```

**Comment by GJRobert** [comment]
A great tool to generate as many colors easily as it allows: ⭐️ [Super Color Palette](https://supercolorpalette.com/)
Very handy tweakings!
For example: 
![色盤](https://github.com/user-attachments/assets/753f3462-0301-4689-bf98-997e33e960a7)
`EAFBD0-D989E1-E189C8-E189A8-E18989-E1A989-E1C989-D8E189-B8E189-98E189-89E19B-89E1BB-89E1DB-89C7E1-89A7E1-8C89E1-AC89E1-CC89E1`

**Comment by 3hr80FEZwlir** [comment]
All colors with clearly noticeable difference between hues.

![Colors](https://github.com/user-attachments/assets/0bcb81a0-bf79-49f2-8312-607ab5012747)

`6C6C6C-302424-B41C1C-EECCC9-F44434-D03522-F4A498-8C6C64-F47C5E-E45010-543C30-F66012-A48C7C-F47C0C-FCB44C-FCA824-FC9C04-FCC61E-FCD44C-F6E490-ECE139-E9E6AE-EAEA66-F8F8C4-B8C030-6C8020-849C30-94C858-346C1C-99BC8C-64B04C-D6E4D6-44A444-348438-64BC6C-22483A-047C6C-84CCC4-0C928C-B0D8D7-205C5C-0494A4-4CD4E4-04BCD4-04ACC4-24C4DC-94A4AC-6C99B1-58BCF4-0F97E7-38ACF4-8ECAF0-5C7484-116EC6-C0D4F4-2048A2-3C54B4-5C6CC4-29268C-4424A4-643CB4-7C54C4-E6E1EE-4C148C-6626A8-A870CC-BE9EC9-9424B0-AC44BC-DEBCD1-B81858-F478A4-EC447C-E62E5E-7C3949`

**Reply by YYY2357** [reply]
Genius!

**Comment by Niszczyciel-ART** [comment]
![image](https://github.com/user-attachments/assets/77d1398c-3ab3-4dd5-b951-fbe82b16a2d5)

`8F052E-D20F32-EB424D-7F1535-BA2641-D55860-71233A-A53B4F-C16C71-8D0724-D01124-E94444-751F29-AB3636-C77066-672D34-954B4B-B3807A-3F2D67-6B4B95-997AB3-32098B-6414CC-9E47E6-381A7A-672EB2-9C5FCE-10078D-3512CE-7645E8-1F187B-452CB5-7F5DD0-2C276D-53429E-8872BB-07298D-122BCE-4548E8-18317B-2C3EB5-5D5FD0-27396D-424E9E-7273BB-045C8F-0D73D3-4185EC-25546F-3F72A2-6F8FBE-335061-54718C-8392AA-03565E-0A84A3-16A3E3-13494E-287386-4193B9-223C3F-43636B-688592-671804-AB3F0D-E6751E-5C1E0F-98461F-CD7837-50251B-844D33-B27C53`

**Comment by derevyan** [comment]
![Screenshot 2025-03-26 at 17 05 58](https://github.com/user-attachments/assets/97248daf-8914-42e2-aaae-e412c4115092)
f94144-f3722c-f8961e-f9844a-f9c74f-90be6d-43aa8b-4d908e-577590-277da1

**Comment by spacetime-labs** [comment]
Here is mine :) 
I use it with the _**high text contrast**_ setting turned on.

73fba9-10ffbb-0271cf-1e3797-09c5d6-35ebde

<img width="466" alt="Screenshot 2025-04-27 at 3 25 59 pm" src="https://github.com/user-attachments/assets/6d121a18-dd6f-4bde-9d2a-ff00bf7ba19d" />

**Comment by VIRGO-AIORIA** [comment]
Some of **AnuPpuccin “Mocha” Theme Colors**
![image](https://github.com/user-attachments/assets/62f3b7eb-f22b-408a-a2c1-59fce5380c00)
f38ba8-eba0ac-fab387-f9e2af-a6e3a1-94e2d5-89dceb-74c7ec-87b0f9-b4beff

Full **AnuPpuccin “Mocha” Theme Colors**
![image](https://github.com/user-attachments/assets/2c1b49e7-5c24-46ed-aab9-02f46a99ca89)
f5e0dc-f2cdcd-f5c2e7-cba6f7-f38ba8-eba0ac-fab387-f9e2af-a6e3a1-94e2d5-89dceb-74c7ec-87b0f9-b4beff

Another Palette
![image](https://github.com/user-attachments/assets/ca965073-3c8f-4651-803c-11d5afcf166d)
e22c3c-e9404b-ee6748-fa9f50-ffd85e-97e768-52eea3-51e1e9-43cfea-54b6f8-437cf3-6f51f4-9446f8-c952ed-e54f9b-e3365e

**Reply by Niszczyciel-ART** [reply]
Mocha is amazing!

**Comment by ellastasia** [comment]
![image](https://github.com/user-attachments/assets/58a1ee7d-6083-42c6-b664-fb0735427213)

b7094c-a01a58-892b64-723c70-5c4d7d-455e89-2e6f95-1780a1-0091ad-006494-003554-051923

**Comment by daeavelwyn** [comment]
<img width="847" height="45" alt="image" src="https://github.com/user-attachments/assets/5a363fd7-5314-447d-9315-49d0140ebbc7" />
480ca8-6e25b8-b5179e-f37324-ff8500-ffc300-ffea00-8ac926-72b043-007f4e-339989-028090-05668d-5e60ce

**Comment by olegarhn** [comment]
07beb8-3dccc7-68d8d6-82e1e3-8fe6e9-9ceaef-c4fff9-c9fffa
<img width="1213" height="119" alt="Screenshot 2025-11-18 at 02 07 38" src="https://github.com/user-attachments/assets/f850091e-0134-4c47-aab2-df35595ada1a" />

**Reply by spacetime-labs** [reply]
It’s interesting how you pretty much downvoted everyone else’s palette.

**Reply by dhniceday** [reply]
True. That's bullying and his posts should be removed.

**Comment by olegarhn** [comment]
FFFFFF-0032A0-DA291C

**Comment by oldtowneast** [comment]
**Flexoki Ultra**

```
d14d41-af3029-da702c-bc5215-d0a215-ad8301-879a39-66800b-3aa99f-24837b-4385be-205ea6-8b7ec8-5e409d-ce5d97-a02f6f
```

![Flexoki-Ultra](https://github.com/user-attachments/assets/a293d0bb-8cc0-436c-9c84-5cffa03519d3)

**Comment by blasmateo** [comment]
Frutal Exclusivo

```
001d41-00dde0-8ce788-f5ff33-ff3f0a-b71112-910041-360b51
``` 
<img width="757" height="101" alt="image" src="https://github.com/user-attachments/assets/3e436243-dcc2-4541-9d38-91dc2e84161a" />

**Comment by kv-obs** [comment]
e3d2e9-f7fe7e-f5ff33-c4aeb1-f3dbda-f9eded-f2f2f2-ecaf92-4ab091-f4dcff

Awesome plugin :)
<img width="2704" height="250" alt="image" src="https://github.com/user-attachments/assets/dafd15f7-3fe3-457d-b1f6-de22869618c4" />

**Reply by j0rdnh** [reply]
Love this palette!

**Comment by upetik** [comment]
Blush 

```
C8636A-B83337-BC8B7B-DCB7AD-C3ACA5-DDA5BB-F1A3CA-F6C8D9-F5CDF4-FCEFFE-C7CEE2-A2ADD0-9880D5-9471D7-B5A3E5-C7BFED-D4CAE9-AFA4C6-4E4956-B7C074-C9E4B3-CCECC2-D5F4B9-E1FEAF-F7FEAF

```

<img width="747" height="51" alt="Screenshot 2026-01-03 at 08 58 29" src="https://github.com/user-attachments/assets/3813643b-88cb-4ecc-87ab-24af09a07496" />


AccentColor: #4A4453

**Comment by bcook128** [comment]
396545-ebdbb2-689d6a-b16286-458588-d79921-98971a-cc241d-282828-09244b-d45916
<img width="766" height="45" alt="Screenshot_20260111_023231" src="https://github.com/user-attachments/assets/da923825-ff48-46eb-aab0-3ae871999b0c" />

**Comment by MianMianChan** [comment]
1e2324-202426-212328-23222a-24222e-232431-222633-232835-242a37-252c39-242c34-232a30
<img width="1160" height="276" alt="image" src="https://github.com/user-attachments/assets/b97a7a93-5ffa-481d-9dea-59d12b44dff9" />

**Comment by to-turion** [comment]
That other notes app ("select" property background)

<img width="1523" height="98" alt="Obsidian 2026-02-12 19 39 32" src="https://github.com/user-attachments/assets/e1b973ba-cadd-4f83-b501-a6e53d49ff95" />

```
F1F1EF-E5E4E1-F0DED7-F8DCCB-F8E6BA-DCEBDD-D5E7F2-E8DDEF-F8DFEB-FBDED9-2F2F2F-373737-613B2C-784722-8A6430-27593E-284B61-4A2F63-672E49-733534
```

**Comment by IHaveNoShame** [comment]
Suitable for those who have read Margulan Seisembai's book Mission Accomplished 2.0

16A34A-D97706-C81E1E-1E40AF-0284C7-4B5563-1E3A8A-1E40AF-1D4ED8-2563EB-3B82F6-60A5FA-EA580C-F97316-FB923C-FDBA74-9333EA-F43F5E-10B981-0EA5E9-F8FAFC-A16207-CA8A04-EAB308-FACC15-FEF08A

<img width="750" height="61" alt="image" src="https://github.com/user-attachments/assets/dd8d7f4f-fe17-46f8-a3b0-cda339ed9575" />

**Comment by annamad2002** [comment]
ffd6ff-e7c6ff-c8b6ff-b8c0ff-bbd0ff

**Reply by annamad2002** [reply]
<img width="909" height="179" alt="Screenshot 2026-03-10 171047" src="https://github.com/user-attachments/assets/6a4f43da-2202-4046-aad5-490f5bbb5349" />

**Comment by DesignwithHugo** [comment]
eddcd2-fff1e6-fde2e4-fad2e1-c5dedd-dbe7e4-f0efeb-d6e2e9-bcd4e6-99c1de

**Reply by DesignwithHugo** [reply]
Got it from [Coolors](https://coolors.co/eddcd2-fff1e6-fde2e4-fad2e1-c5dedd-dbe7e4-f0efeb-d6e2e9-bcd4e6-99c1de) website, not my original selection. 
<img width="462" height="163" alt="Screenshot 2026-03-16 at 10 02 58 p m" src="https://github.com/user-attachments/assets/6e5f0a24-a0a5-4d57-836c-e5c5318098d6" />

**Comment by MikeshCZ** [comment]
As the right theme selector it must contain [Catppuccin](https://catppuccin.com/) pallete 🙂

**Frappé** 
```
f2d5cf-eebebe-f4b8e4-ca9ee6-e78284-ea999c-ef9f76-e5c890-a6d189-81c8be-99d1db-85c1dc-8caaee-babbf1
```

**Mocha**
```
f5e0dc-f2cdcd-f5c2e7-cba6f7-f38ba8-eba0ac-fab387-f9e2af-a6e3a1-94e2d5-89dceb-74c7ec-89b4fa-b4befe
```

**Macchiato** 
```
f4dbd6-f0c6c6-f5bde6-c6a0f6-ed8796-ee99a0-f5a97f-eed49f-a6da95-8bd5ca-91d7e3-7dc4e4-8aadf4-b7bdf8
```

**Latte** 
```
dc8a78-dd7878-ea76cb-8839ef-d20f39-e64553-fe640b-df8e1d-40a02b-179299-04a5e5-209fb5-1e66f5-7287fd
```

**Reply by cyhyraethz** [reply]
Nice work! These should definitely be included in the list of community pallets on the plugin page. It's exactly what I was looking for and would have saved me some time coming up with the same thing for my favorite Catppuccin Macchiato theme. Love that you shared all the flavors.

**Comment by jhilker98** [comment]
Gruvbox my beloved

**Normal Colors**
`cc421d-98971a-d79921-458588-b16286-689d6a-d65d0e`
**Bright Colors**
`fb4934-b8bb26-fabd2f-83a598-d3869b-8ec07c-fe8019`

**Comment by Badgov** [comment]
Yo uso esta paleta de colores, más saturados:

<img width="774" height="47" alt="image" src="https://github.com/user-attachments/assets/9d772533-587e-4a0a-a691-f97e0b4e005b" />

c1121f-d62828-e85d04-f77f00-f0a500-5a7d09-2dc653-059c3a-007200-008000-00b4d8-0096c7-0077b6-023e8a-4361ee-3a0ca3-7209b7-560bad-9d0208-6a0572

**Comment by Mozgoder69** [comment]
<img width="928" height="40" alt="image" src="https://github.com/user-attachments/assets/1b603af6-2903-4796-92bc-5c3e58b92330" />

```
cc4444-cc8800-cccc44-44cc44-339999-006699-333399-993399
```

**Comment by voidinthetrenchcoat** [comment]
Viridis    
`fde725-b5de2b-6ece58-35b779-1f9e89-26828e-31688e-3e4989-482878-440154`
<img width="750" height="35" alt="image" src="https://github.com/user-attachments/assets/37f39db5-de59-48a7-aedb-7831437a8526" />

Viridis to plasma  
 `440154-43236e-414487-365e8b-2a788e-269089-22a884-4ebd6b-7ad151-bcdc3b-fde725-f0f921-f6d02c-fca636-ef854c-e16462-c94779-b12a90-8e159c-6a00a8-3c0498-0d0887`
<img width="750" height="38" alt="image" src="https://github.com/user-attachments/assets/eb125f20-e4c1-42db-b6d9-16c2b87cae5a" />

**Comment by lanius-mentis** [comment]
<img width="410" height="50" alt="Obsidian_UFZ7ueIWGH" src="https://github.com/user-attachments/assets/a6ca7277-5616-4528-85b5-81e5b39e3fee" />

```
668822-cc9933-cc6633-cc3366-884488-6633cc-3366cc-3399cc-228866
```

**Comment by kchen0567** [comment]
fbf8cc-fde4cf-ffcfd2-f1c0e8-cfbaf0-a3c4f3-90dbf4-8eecf5-98f5e1-b9fbc0

**Comment by nipunbatesman-ux** [comment]
A Dark Academia/Brown-ish palette:

d4c4ad-c7b299-b8a58d-a89278-9c7b63-8f7867-826b5b-7b5c49-706864-6b5a4d-66584e-5e4434-5b5450-524841-4a4039-3f3834-b8956a-a98558-9a7a52-8a6748-7c6145-6f563d-d2c0aa-cbb49a-b29a82

**Comment by crmills81** [comment]
Just Pastel colors all over the place! PASTEL ALL THE THINGS!!!!


<img width="1326" height="62" alt="image" src="https://github.com/user-attachments/assets/82300ac1-64ac-449d-9292-0bad5b74ba87" />


d4cfff-ffe0e2-ffd29c-ffd0f2-cff2ff-ffeaef-decfff-ffcffa-decfff-ffe0e2-ffd29c-ffd0f2-bfd4ff-ffeaef-decfff-ffcffa-bfd4ff-ffe0e2-ffd29c-ffd0f2-bfd4ff-cff2ff-decfff-ffcffa

**Comment by yugofox** [comment]
a kind of frosty-steel palette with yellow for things i gotta remember or just rlly like c:


<img width="469" height="59" alt="frosty-colors" src="https://github.com/user-attachments/assets/5357cc8e-2ca6-4aee-be3e-2b70be8d3c15" />

e8f1f7-bfd7ea-7ea3cc-4f6d8c-1f3044-b8c4ce-ffd84d

**Comment by definitely-not-flo** [comment]
<img width="763" height="40" alt="image" src="https://github.com/user-attachments/assets/8027a6a2-f85d-47f5-9bd4-0a5934354d5b" />

`EC8990-EA999C-ED877F-FF8964-F49D71-49C1C1-59BAC8-5CBAE4-75B5F0-77ABE6-87B0FC-8CA9F2-9E9EF5-A6A7F2-B3A2E9-C7A6E5-C58DD7-D3A6CD-CB8DBB-EC95D1-EB8ACB-E791B7-EC95BA-EB8AB2-E67B93-E78284-EE7A6A-EA7956-EA8044-7BAA46-6EA94C-68B464-56B362-64B57F-66B18A-3CBB97-32B6A5-48ABC5-5DABE4-73A4E8-889CE8-B47DD4-C377CA-C585BF-C07DB3-CB7EB2-C585A9-CB7E9E-AE375F-C76B7C-BD576A-D5687A-CE5D70-C8354E-E66772-E46666-D36A55-5F9A42-639F56-389F7C-559E86-307FA6-4B77CF-7E8BE7-6C6CD0-8981E4-8D7CDE-774BCF-8E64CE-883DAE-A764A6-963D83-C577AA-CF4BA3-9C7177-8B6268-785057-9F7873-8E6763-7B5653-9F7873-8E6763-7B5653-97777A-87666A-74555A-95777B-84676B-71565A-907782-7F6772-6C5660-41393C-534A5A`

**Comment by PeonyPicker** [comment]
<img width="568" height="37" alt="Screenshot 2026-07-18 224514" src="https://github.com/user-attachments/assets/501148a6-1f21-4abe-a22b-69f9d99c3c36" />

e76f51-ee8959-f4a261-efb366-e9c46a-BABB74-8ab17d-2a9d8f-287271-264653

**Comment by Ichigo-Meow** [comment]
My [Eldritch](https://github.com/eldritch-theme) palette

<img width="746" height="34" alt="image" src="https://github.com/user-attachments/assets/62f1a270-17a1-49f1-a87a-fa3441f40604" />

```
ebfafa-7081d0-04d1f9-37f499-f7c67f-f265b5-a48cf2-f16c75-f1fc79
```
